content/2.0.html.haml

```diff
@@ -90,109 +89,115 @@ title: "Jenkins 2.0"
 
 %h2
   Highlights
+%ul 
+  %li Jenkins 2.0 supports delivery pipelines natively. 
+  %li Jenkins 2.0 improves usability.
+  %li Jenkins 2.0 is backwards compatible.
 
-%dl#highlights
-  %dt
-    %strong
-      Pipeline as Code
-    keeps delivery
-    %a{:href => 'https://en.wikipedia.org/wiki/Don%27t_repeat_yourself'}
-      DRY
-  %dd
-    %p
-      The new
-      %a{:href => '/solutions/pipeline'}
-        Pipeline
-      functionality in Jenkins allows you to define configuration as code,
-      which can be checked in and version controlled along with the rest of
-      your project's source code.
-
-    %img{:src => '/images/hello-world-pipeline.png', :title => 'A simple build/test pipeline'}/
-
-    %p
-      Defining your pipeline's configuration as code makes it easier to create
-      a simple "build and test" pipeline, while enabling more advanced and
-      complex pipelines through the expressive Groovy-based domain specific
-      language.
-
-    %p
-      With the new Pipeline Stage View Plugin, the different stages of pipelines
-      can easily be visualized.
-
-    %a{:href => '/images/pipeline-stage-view.png', :target => '_blank'}
-      %img{:src => '/images/pipeline-stage-view.thumb.png', :title => 'A pipeline stage view',
-          :width => 550}/
-
-    %p
-      Learn more about this on our
-      %a{:href => '/solutions/pipeline'}
-        Pipeline solutions page
-      !
-
-  %dt
-    Out of the box: Easy to get it right
-  %dd
-    %p
-      For new users, Jenkins 2.0 starts off with set of recommended plugins to
-      help get you started with the right set of tools to get up and running with
-      Jenkins quickly.
-
-      For the more adventurous users, the Jenkins 2.0 initial setup process
-      also allows you to pick and choose exactly the plugins you want to meet
-      your specific needs. The list of plugins is hand-picked by Jenkins experts
-      from our popular plugins to ensure the best getting-started experience for
-      new users.
-
-    %a{:href => '/images/getting-started-setup.png', :target => '_blank'}
-      %img{:src => '/images/getting-started-setup.png', :title => 'Getting started with Jenkins 2.0',
+%h2
+  Pipelines
+
+%h3 Problem
+
+%p As organizations of all types seek to deliver high quality software faster, their use of Jenkins is extending beyond just CI to Continuous Delivery (CD) In order to implement CD, teams  need a flexible way to model and orchestrate their entire delivery pipeline.
+  
+%h3 Solution
+
+%p Jenkins 2.0 now natively supports delivery pipelines as a first-class entity. Pipelines introduces a domain specific language that help Jenkins users to model their software delivery pipelines as code, which can be checked in and version controlled along with the rest of your project's source code.
```

Comment:
now natively supports -> will support (I'd remove *natively* as it's covered by being a *first-class entity*).

Pipeline introduces -> Pipelines introduce

checked in and version controlled -> version-controlled

rest of your project's -> rest of their project's (you switched from third person to first person)